src/core_plugins/kibana/server/routes/api/values/register_values.js

```diff
@@ -0,0 +1,57 @@
+import handleESError from '../../../lib/handle_es_error';
+
+export function registerValues(server) {
+  server.route({
+    path: '/api/kibana/values/{index}',
+    method: ['POST'],
```

Comment:
Please define validation rules for the supported params and payload parameters. I think it should look something like:

```
server.route({
```